README.md

```diff
@@ -363,10 +364,28 @@ If you need a higher limit, drop us a line at support@scalyr.com.
 
 ## Fetching numeric data using a timeseries
 
-The "timeseries-query" command allows you to retrieve numeric data using a timeseries defined using the
-create-timeseries command. (Note that the [Scalyr API](https://www.scalyr.com/help/api#timeseriesQuery)
+A timeseries precomputes a numeric query, allowing you to execute queries almost instantaneously, and without
+exhausting your query execution limit.  This is especially useful if you are using the Scalyr API to feed a
```

Comment:
`s/exhausting your query execution limit/consuming your account's query budget/`

("exhaust" is a strong word!)